js/build.js

```diff
@@ -1,14 +1,17 @@
 'use strict';
 
 var entries = [];
+var projWriteup = [];
 
+// CONSTRUCTOR FUNCTION FOR JOB EXPERIENCE OBJECTS TO BE USED FOR RENDING WITH JQUERY TO HTML PAGE
```

Comment:
I like the use of 'options' passed into this constructor a lot better than the 'opts' that we did in class. I will likely follow your example and start doing this. opts just doesn't intuitively appear as 'options' to me when I read it and muddies up my code..